assignments/arrays.js

```diff
@@ -62,33 +62,60 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // }
 
 // ==== Challenge 1 ====
-// The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
+// The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which 
+//car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
+
 console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+for (let i = 0; i < inventory.length; i++){
+    if (inventory[i].id === 33){
+        console.log(`Car 33 is a ${inventory[i].car_year} ${inventory[i].car_make} ${inventory[i].car_model}`);
```

Comment:
Elyssia in this challenge you did not need to use a for loop, even though your for loop works and gets the object with the `id` of 33 there is a faster and easier way to approach this problem like so -> 

```
console.log("Car 33 is a " + inventory[32].car_year + " " + inventory[32].car_make + " " + inventory[32].car_model);
```
or
```
console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`);
```